Misc/Adam.cpp

```diff
@@ -3354,3 +3354,24 @@ int AdamInit()
 
 	return 1;
 }
+
+void WriteAlteredConfigVarsToIni()
+{
+	const char* alteredConfigVars[] = {
+		"projmetroen",
+		"relativeedges",
```

Comment:
`relativeedges` is never modified by SWS: the `AWRelEdge*` actions were removed in 1ad238265873b71faa14b4e0821b5e901c1b245b. Those unused functions should probably be deleted...